### PR TITLE
Fluent-bit upgrade to v2.0.11

### PR DESCRIPTION
**Pull request by wenchajun**
### Which issue(s) this PR fixes:

Fixes #

### Does this PR introduced a user-facing change?

```release-note

```

### Additional documentation, usage docs, etc.:

```docs

```

### CODE REVIEW

File: charts/fluent-operator/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ description: A Helm chart for Kubernetes
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.1.1
+version: 2.1.2
```
Comment (benjaminhuo) [inline charts/fluent-operator/Chart.yaml:18]: version can be the same as 2.2.0

Comment (wenchajun) [reply charts/fluent-operator/Chart.yaml:18]: Should I wait for version 2.2.0 to be released and then update the field to 2.2.0?

Comment (benjaminhuo) [reply charts/fluent-operator/Chart.yaml:18]: we're going to release v2.2.0 after this PR, so it's ok to change it now

Comment (wenchajun) [reply charts/fluent-operator/Chart.yaml]: fixed

**Review by benjaminhuo — APPROVED** [review]

**Pull request merged**